### PR TITLE
Singulo Equipment Cleanup and Fixes

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Just some singulo equipment cleanup and fixes

- Cleans out some old, unused code for R-UST regarding PA particles
- Fixes hacked overloaded PA's doing nothing
- Fixes admins not being notified when the singularity escapes
- Fixes some construction behavior when attempting to build the PA in space or while stunned
- Adds in Destroy()'s for the PA and Control Box

### CODE REVIEW

File: code/modules/power/singularity/field_generator.dm [hunk 1]
```diff
@@ -344,10 +344,10 @@ field_generator power level display
 	//I want to avoid using global variables.
 	spawn(1)
 		var/temp = 1 //stops spam
-		for(var/obj/singularity/O in machines)
+		for(var/obj/singularity/O in world)
```
Comment (tigercat2000) [inline code/modules/power/singularity/field_generator.dm]: why would you ever change this to an `in world` call
make a new global list for singularities or just keep adding them to machines, don't use `in world`, that's horrid

File: code/modules/power/singularity/particle_accelerator/particle_accelerator.dm [hunk 2]
```diff
@@ -206,35 +216,39 @@ So, hopefully this is helpful if any more icons are to be added/changed/wonderin
...
 					"You remove some wires.")
 				temp_state--
-			else if(isscrewdriver(O))
+			else if(istype(O, /obj/item/weapon/screwdriver))
 				user.visible_message("[user.name] closes the [src.name]'s access panel.", \
 					"You close the access panel.")
 				temp_state++
 		if(3)
-			if(isscrewdriver(O))
+			if(istype(O, /obj/item/weapon/screwdriver))
```
Comment (tigercat2000) [inline code/modules/power/singularity/particle_accelerator/particle_accelerator.dm]: why would you change these
we have those helper procs for specifically this reason, everything should be using them, not a direct istype

File: code/modules/power/singularity/particle_accelerator/particle_control.dm [hunk 3]
```diff
@@ -233,9 +239,8 @@
 	user.set_machine(src)
 
 	var/dat = ""
-	dat += "Particle Accelerator Control Panel<BR>"
 	dat += "<A href='?src=\ref[src];close=1'>Close</A><BR><BR>"
-	dat += "Status:<BR>"
+	dat += "<h3>Status</h3>"
```
Comment (tigercat2000) [inline code/modules/power/singularity/particle_accelerator/particle_control.dm]: this is going to end with
### Status Unable to detect all parts!
[Run Scan]()

**Comment by ZomgPonies** [comment]
:-1: world calls

**Comment by Fox-McCloud** [comment]
Fixed up both things.

I still hate tool helpers.

**Comment by tigercat2000** [comment]
@Fox-McCloud The particle_control HTML is still going to have a problem, it has no line break between `<h3>Status</h3>` and the next part.

**Comment by Fox-McCloud** [comment]
only...it's fine:

**Comment by tigercat2000** [comment]
... okay then.

**Pull request merged**